recipes-devtools/gie/tensorrt-plugins_8.0.1-1.bb

```diff
@@ -0,0 +1,52 @@
+DESCRIPTION = "NVIDIA TensorRT Plugins for deep learning"
+HOMEPAGE = "http://developer.nvidia.com/tensorrt"
+LICENSE = "Apache-2.0"
```

Comment:
There are more licenses than just Apache in the included code, from what I can tell.